content/docs/releases/release-notes/release-notes-1.15.md

```diff
@@ -0,0 +1,366 @@
+---
+title: Release 1.15
+description: 'cert-manager release notes: cert-manager 1.14'
+---
+
+> 📢 The cert-manager CLI has moved to a new GitHub repository
+>
+> From this release, `cmctl` is no longer be released with `cert-manager` itself,
+> and there will no further `quay.io/jetstack/cert-manager-ctl` OCI images.
+>
+> Read [The cert-manager Command Line Tool (cmctl) page](../../reference/cmctl.md) to learn more.
+
```

Comment:
Should we also add a note about the CRDs which are no longer uninstalled when you uninstall using Helm?